exonum-java-binding/core/rust/src/storage/db.rs

```diff
@@ -55,7 +55,7 @@ impl View {
         View {
             // Make a "self-reference" to a value stored in the `owned` field.
             reference: unsafe { ViewRef::from_snapshot(&*snapshot) },
-            _owned: Some(ViewOwned::Snapshot(snapshot)),
+            owned: Some(ViewOwned::Snapshot(snapshot)),
```

Comment:
Why initialization order is not consistent with the declaration order?

Reply:
Also, see the comment in #1048 regarding the non-owned views and service runtimes.

Reply:
This initialization order is needed for correct borrowing usage, because we need to take pointer before moving snapshot into the `owned`

Reply:
I see, that was unexpected :-)